spec.html

```diff
@@ -98,7 +98,7 @@ <h1>[[Call]] ( _thisArgument_, _argumentsList_ )</h1>
 			1. Let _wrappedThisArgument_ to ? GetWrappedValue(_targetRealm_, _thisArgument_).
 			1. Let _result_ be the Completion Record of Call(_target_, _wrappedThisArgument_, _argumentsList_).
 			1. If _result_.[[Type]] is ~normal~ or _result_.[[Type]] is ~return~, then
-				1. Set _value_ to NormalCompletion(_result_.[[Value]]).
+				1. Set _value_ to _result_.[[Value]].
```

Comment:
```suggestion
```
this step can be removed entirely, since completion records get implicitly converted to have `.[[Value]]`